docs/services/slam/cartographer/_index.md

```diff
@@ -330,8 +330,8 @@ Adjust these parameters to fine-tune the algorithm `cartographer` utilizes in as
 | `optimize_every_n_nodes` | How many trajectory nodes are inserted before the global optimization is run. | Optional | `3` | |
 | `num_range_data` | Number of measurements in each submap. | Optional | `100` | |
 | `missing_data_ray_length` | Replaces the length of ranges that are further than `max_range` with this value. | Optional | `25` | Nominally set to max length. |
-| `max_range` | Maximum range of valid measurements. | Optional | `25` | |
-| `min_range` | Minimum range of valid measurements. | Optional | `0.2` | |
+| `max_range` | Maximum range of valid measurements. | Optional | `25` | The default value is for RPLIDAR A3. For RPLIDAR A1, use `12`. |
+| `min_range` | Minimum range of valid measurements. | Optional | `0.2` | The default value is for RPLIDAR A3. For RPLIDAR A1, use `0.15`. |
```

Comment:
[nit/opt] I would keep it more open without saying that the value is specified for the Rplidar A3 (since it could be true for other sensors, too). Something like this?

```suggestion
| `max_range` | Maximum range of valid measurements. | Optional | `25` | For an RPLIDAR A3, set this value to `25`. For an RPLIDAR A1, use `12`. |
| `min_range` | Minimum range of valid measurements. | Optional | `0.2` | For an RPLIDAR A3, set this value to `0.2`. For RPLIDAR A1, use `0.15`. |
```

Reply:
Sounds good! Thanks for the suggestion.